src/main/java/uk/gov/hmcts/reform/pdf/service/endpoint/v2/PDFGenerationEndpointV2.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.pdf.service.endpoint.v2;
+
+import io.swagger.annotations.Api;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pdf.generator.HTMLToPDFConverter;
+import uk.gov.hmcts.reform.pdf.service.domain.GeneratePdfRequest;
+import uk.gov.hmcts.reform.pdf.service.domain.GeneratePdfResponse;
+
+@Api
+@RestController
+@RequestMapping(
+    path = "pdf",
+    produces = PDFGenerationEndpointV2.MEDIA_TYPE
+)
+public class PDFGenerationEndpointV2 {
+
+    public static final String MEDIA_TYPE = "application/vnd.uk.gov.hmcts.pdf-service.v2+json";
```

Comment:
if this is meant to be input type, then it may be worth including 'twig' or 'html' in type name.